### ISSUE TITLE
composter is registered as visual effect 2.6.2

### ISSUE THREAD

**Issue by PotatoIQ** [SkriptLang/Skript#4773]
### Skript/Server Version

```markdown
28.05 06:18:16 [Server] INFO [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
28.05 06:18:16 [Server] INFO [Skript] Skript's documentation can be found here: https://skriptlang.github.io/Skript
28.05 06:18:16 [Server] INFO [Skript] Server Version: git-Paper-350 (MC: 1.18.2)
28.05 06:18:16 [Server] INFO [Skript] Skript Version: 2.6.2
28.05 06:18:16 [Server] INFO [Skript] Installed Skript Addons: 
28.05 06:18:16 [Server] INFO [Skript]  - skript-placeholders v1.5.2 (https://github.com/APickledWalrus/skript-placeholders)
28.05 06:18:16 [Server] INFO [Skript]  - Floodgate-Skript v2.1
28.05 06:18:16 [Server] INFO [Skript]  - Skellett v2.0.3 (https://forums.skunity.com/resources/skellett.24/)
28.05 06:18:16 [Server] INFO [Skript]  - skRayFall v1.9.27 (https://sk.rayfall.net/)
28.05 06:18:16 [Server] INFO [Skript]  - SkBee v1.15.3 (https://github.com/ShaneBeee/SkBee)
28.05 06:18:16 [Server] INFO [Skript] Installed dependencies: 
28.05 06:18:16 [Server] INFO [Skript]  - SQLibrary v7.1
28.05 06:18:16 [Server] INFO [Skript]  - Vault v1.7.3-b${env.TRAVIS_BUILD_NUMBER}
28.05 06:18:16 [Server] INFO [Skript]  - WorldGuard v7.0.7+216b061
28.05 06:18:16 [Server] INFO [Skript]  - GriefPrevention v16.18
```

### Bug Description

```
on right click with bamboo:
    if targeted block is composter:
on right click with bamboo:
    if clicked block is composter:
on right click with bamboo:
    if event-block is composter:
on right click with bamboo:
    if block at event-location is composter:
```
All of these gives the error:
Can't compare a block with a visual effect

It was working fine in 2.6.1, with same addons. only updated skript to 2.6.2 and it started happening

### Expected Behavior

it should let me compare composter to clicked block

### Steps to Reproduce

write this:
```
on right click with bamboo:
    if targeted block is composter:
on right click with bamboo:
    if clicked block is composter:
on right click with bamboo:
    if event-block is composter:
on right click with bamboo:
    if block at event-location is composter:
```

### Errors or Screenshots

```
Can't compare a block with a visual effect
Line: if targeted block is composter:
```
```
Can't compare a block with a visual effect
Line: if clicked block is composter:
```
```
Can't compare a block with a visual effect
Line: if event-block is composter:
```
```
Can't compare a block with a visual effect
Line: if block at event-location is composter:
```

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by TheLimeGlass** [comment]
This was temporarily fixed in 2.6.3, but didn't have a complete solution. The block itemtype in your case will make your code work, but as for using composter as it's particle effect it has, it would be `composter particle`

**Comment by TheLimeGlass** [comment]
I have an upcoming fix
